FirebaseAuth/Sources/Swift/Auth/Auth.swift

```diff
@@ -1667,6 +1659,23 @@ extension Auth: AuthInterop {
    This case will return `nil`.
    Please refer to https://github.com/firebase/firebase-ios-sdk/issues/8878 for details.
    */
+  @objc(getStoredUserForAccessGroup:error:)
+  public func __getStoredUser(forAccessGroup accessGroup: String?,
+                              error outError: NSErrorPointer) -> User? {
```

Comment:
If you'd like to make this API inaccessible for Swift clients, and only available via Objective-C, add `@available(*, unavailable)` before the `@objc` attribute. 

```suggestion
  @available(*, unavailable)
  @objc(getStoredUserForAccessGroup:error:)
  public func __getStoredUser(forAccessGroup accessGroup: String?,
                              error outError: NSErrorPointer) -> User? {
```

Reply:
@ncooke3 This isn't working. From https://github.com/firebase/firebase-ios-sdk/actions/runs/6946785601/job/18899104418#step:7:58:

```
    - ERROR | [tvOS] xcodebuild:  /Users/runner/work/firebase-ios-sdk/firebase-ios-sdk/FirebaseAuth/Tests/Unit/ObjCAPITests.m:200:31: error: 'getStoredUserForAccessGroup:error:' is unavailable
    - NOTE  | [tvOS] xcodebuild:  /Users/runner/Library/Developer/Xcode/DerivedData/App-ayiyfckbnyudiofjbmlqimvaarkn/Build/Products/Debug-appletvsimulator/FirebaseAuth/FirebaseAuth.framework/Headers/FirebaseAuth-Swift.h:862:1: note: 'getStoredUserForAccessGroup:error:' has been explicitly marked unavailable here
```

Reply:
Oh, sorry for misleading. You might be able to instead do:
```suggestion
  @available(swift 100.0)
  @objc(getStoredUserForAccessGroup:error:)
  public func __getStoredUser(forAccessGroup accessGroup: String?,
                              error outError: NSErrorPointer) -> User? {
```